colors/dracula.vim

```diff
@@ -431,15 +433,16 @@ hi! link rubyGlobalVariable DraculaPurple
 hi! link rubyRegexpDelimiter DraculaRed
 
 "}}}2
-" YAML: {{{2
+" Sass: {{{2
 
-hi! link yamlBlockMappingKey DraculaCyan
-hi! link yamlPlainScalar DraculaYellow
-hi! link yamlAnchor DraculaPinkItalic
-hi! link yamlAlias DraculaGreenItalicUnderline
-hi! link yamlNodeTag DraculaPink
-hi! link yamlFlowCollection DraculaPink
-hi! link yamlFlowIndicator Delimiter
+hi! link sassProperty cssProp
+hi! link sassMixinName Function
+hi! link sassClass DraculaGreen
+hi! link sassClassChar DraculaGreen
+hi! link sassInterpolationDelimiter DraculaPink
+hi! link sassVariableAssignment DraculaPink
+hi! link sassId DraculaGreen
```

Comment:
Similarly, what about the `Identifier` group? Also appears to be a different color...